Shopify/cli
# Bump/theme check 1.15.0

### WHY are these changes introduced?


This PR bumps Theme Check to the latest release https://github.com/Shopify/theme-check/releases/tag/v1.15.0 to introduce:
- Improve rule for lazy loading to prevent developers from overusing it [#723](https://github.com/Shopify/theme-check/issues/723)
- Introduce `--update-docs` flag to synchronously update Theme Check resources (objects, filters, and tags) [#707](https://github.com/Shopify/theme-check/issues/707)

### WHAT is this pull request doing?

This PR bumps Theme Check to the latest release and introduces a new Theme Check flag (introduce by [#707](https://github.com/Shopify/theme-check/issues/707)).

### How to test your changes?

- Go to a theme directory
- Run `shopify theme check --update-docs`
- Notice that Theme Check downloads docs 
- Notice that Theme Check shows lazy loading warns following the new standards

### Post-release steps

Update [Theme Check docs](https://shopify.dev/docs/themes/tools/theme-check/commands#flags) with the new flag.

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
